### DISCUSSION TITLE
How to transcribe multilangual speech properly. Model bevaviour changes unexpectedly.

### DISCUSSION THREAD

**Discussion by nathbhargav** [openai/whisper#1588] [General]
I have a mix speech of "Hindi" and "English". When I passed through the model it should transcribe "Hindi" part in "Hindi" language and "English" part in English language. But whisper model has shown wired behaviour , sometimes model runs all in English, sometimes it translate to english , sometimes it hallucinate in Hindi and sometime it produces Hindi and English both.

How do I control this behaviour??

**Comment by phineas-pta** [comment]
whisper doesnt support multilingual audio

**Reply by nathbhargav** [reply]
Can I make my whisper model to works like that by doing any techniques like  fine-tuning and all? 
Thank you for your response @phineas-pta
